content/en/docs/setup/install/operator/index.md

```diff
@@ -93,6 +93,26 @@ EOF
 The controller will detect the `IstioOperator` resource and then install the Istio
 components corresponding to the specified (`demo`) configuration.
 
+If you have used `--watchedNamespaces` to initialize the Istio operator, make sure the IstioOperator resource is located in the specified namespace list.
```

Comment:
```suggestion
{{< warning >}}
If you have used `--watchedNamespaces` when you initialized the Istio operator, apply the `IstioOperator` resource in one of the watched namespaces, instead of in `istio-system`.
{{< /warning >}}
```